src/View/InstallQuickInput.ts

```diff
@@ -201,11 +200,12 @@ class InstallQuickInput {
       this.toolchainType = state.selectedItem.label;
     }
 
-    this.changeCurrentStepAfter(this.pickType.name, state as State);
+    this.changeCurrentStepAfter(this.pickType.name, state as InstallQuickInputState);
   }
 
-  async pickVersion(input: MultiStepInput, state: Partial<State>) {
-    this.changeCurrentStepBefore(this.pickVersion.name, state as State);
+  /* istanbul ignore next */
+  public async pickVersion(input: MultiStepInput, state: Partial<InstallQuickInputState>) {
```

Comment:
I think some of codes can be tested, can I ask you why this function is excluded? :)